docs/shared_parsers_catalog/getconf_page_size.rst

```diff
@@ -0,0 +1,3 @@
+.. automodule:: insights.parsers.getenforce
```

Comment:
@chawlanikhil24 the module name needs to match your parser module, `insights.parsers.getconf_pagesize`.